tests/ctst/steps/cloudserverAuth.ts

```diff
@@ -0,0 +1,83 @@
+import { When, Then } from '@cucumber/cucumber';
+import Zenko from '../world/Zenko';
+import { Utils, S3, Constants } from 'cli-testing';
+import { strict as assert } from 'assert';
+
+interface DeleteObjectsResult {
+    Deleted?: {
+        Key: string;
+        DeleteMarker?: boolean;
+        DeleteMarkerVersionId?: string;
+    }[];
+    Errors?: {
+        Key: string;
+        VersionId?: string;
+        Code: string;
+        Message: string;
+    }[];
+}
+
+When('the user tries to perform DeleteObjects', async function (this: Zenko) {
+    this.resetCommand();
+    this.resumeRootOrIamUser();
+    this.addCommandParameter({ bucket: this.getSaved<string>('bucketName') });
+    let keys = '';
+    const objectNames = this.getSaved<string[]>('objectNameArray');
+    objectNames.forEach((objectName, i) => {
+        let key = `{Key=${objectName}}`;
+        if (i < objectNames.length - 1) {
+            key += ',';
+        }
+        keys += key;
+    });
+    this.addCommandParameter({ delete: `Objects=[${keys}]` });
+    this.setResult(await S3.deleteObjects(this.getCommandParameters()));
+});
+
+When('the user tries to perform CreateBucket', async function (this: Zenko) {
+    this.resetCommand();
+    this.resumeRootOrIamUser();
+    const preName = (this.parameters.AccountName || Constants.ACCOUNT_NAME);
+    const usedBucketName = `${preName}${Constants.BUCKET_NAME_TEST}${Utils.randomString()}`.toLocaleLowerCase();
+    this.addToSaved('bucketName', usedBucketName);
+    this.addCommandParameter({ bucket: usedBucketName });
+    this.addCommandParameter({ objectLockEnabledForBucket: ' ' });
```

Comment:
Maybe worth adding a comment saying that it's a way to pass empty parameters, maybe that's not clear for everyone when working on the project